packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -107,6 +107,65 @@ useful properties are listed below:
 <ApiSnippet url={'/api/2d/components/Layout#justifyContent'} />
 <hr />
 
+### Disabling `layout` on a child node
+
+Sometimes it can be useful to remove the node from the node tree. This way the
+node can be styled independently of its ancestor layout, as well as moved freely
+on the canvas.
+
+Removal of node from the node tree is done by setting a value of `layout`
+property to `false`. The node can be brought back to its original position
+inside the node tree by setting the value of `layout` property to `true`.
+
+After disabling `layout` on a node, the following things change:
+
+1. the node is moved to the place of origin, e.g. center of the scene,
+2. the node can now be position on canvas freely,
+3. the node no longer inherits properties from its parent, such as text styling,
+4. the node's former parent layout will no longer considers the node's width or
+   height when calculating its own bounding box. Disabling a layout on a node
```

Comment:
The layout system calculates the sizes and positions of nodes, not their bounding boxes. 
"Bounding box" has a specific meaning in Motion Canvas which is different from the size of a node.